src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx

```diff
@@ -107,24 +113,12 @@ export class ConnectedApps extends Component {
         ))}
 
         {showHasNoConnectedApps && (
-          <div className="connected-apps-intro vads-u-margin-bottom--2">
-            <h3>Third-party apps you can connect to your profile</h3>
-            <ul className="vads-u-padding-left--0 vads-u-margin-bottom--2">
-              {availableConnectedApps?.map(app => {
-                return (
-                  <li key={app.name} className="vads-u-padding-left--3">
-                    <a
-                      href={app.appURL}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      {app.name}
-                    </a>
-                  </li>
-                );
-              })}
-            </ul>
-          </div>
+          <Link
```

Comment:
Just curious why you are using a `Link` here but previously used `a` elements? Also, does it make a difference if we use the `Link` from old `react-router` like you are doing or if we should use the one from `react-router-dom`?